.github/workflows/codeql.yml

```diff
@@ -0,0 +1,40 @@
+name: "CodeQL"
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+    branches:
+      - main
+      - develop
+      - master # for safety reasons
+      - dev # for safety reasons
+
+jobs:
+  analyse:
+    name: Analyse
+    runs-on: ${{ github.repository_owner == 'zondax' && 'zondax-runners' || 'ubuntu-latest' }}
+    if: github.event.repository.private == false
+    strategy:
+      matrix:
+        sdk: ["$NANOX_SDK", "$NANOSP_SDK", "$STAX_SDK", "$FLEX_SDK"]
+    container:
+      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-legacy:latest
```

Comment:
The `ledger-app-builder-legacy` should not be used anymore, you should migrate to either the `ledger-app-builder-lite` or `ledger-app-builder`.